src/AidSelectionStep/use-missing-aid.ts

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
+import { inProgressState, useAnonymizer } from '../shared';
+import { ComputedData, TableSchema } from '../types';
+
+export function useMissingAid(schema: TableSchema, aidColumn: string): ComputedData<boolean> {
+  const anonymizer = useAnonymizer();
+  const [result, setResult] = useState<ComputedData<boolean>>({ state: 'completed', value: false });
+
+  useEffect(() => {
+    if (!aidColumn) {
+      // For no AID we know immediately that there are no NULL values.
+      setResult({ state: 'completed', value: false });
+      return;
+    } else {
```

Comment:
Redundant else indentation because of early return

Reply:
that's kinda deliberate... I've developed strong dislike towards early returns. But if that's too unidiomatic, I'll adjust

Reply:
Both `return` and `else` are redundant. Personally I'd go with less indentation, but feel free to go with either.